scex-util/src/main/scala/com/avsystem/scex/util/function/StringUtil.java

```diff
@@ -48,4 +48,6 @@
     String replace(String str, String find, String replacement);
 
     String join(Collection<String> list, String separator);
+
+    String hmacMd5(String str, String key);
```

Comment:
Cosmetic stuff but `Md5` looks somewhat weird... `MD5`?

Reply:
ok, I'll change it

Reply:
I think you may also want to add this method into `StringMiscOps` (extensions methods for `String` type). The `StringUtil` interface and implementation is somewhat legacy as it comes from very old days of SpEL (Spring Expression Language).

Reply:
thank you for the suggestion, I added it

Reply:
oh, I see I need to run the tests